angular2-jwt.ts

```diff
@@ -211,8 +211,8 @@ export class JwtHelper {
     let decoded: any;
     decoded = this.decodeToken(token);
 
-    if (typeof decoded.exp === 'undefined') {
-      return new Date(0);
+    if (!decoded.hasOwnProperty('exp')) {
```

Comment:
And I believe both are convoluted vs a simple `!decoded.exp` truthy check. Unless setting it to `null` is a valid different value (which makes the undefined check understandable), or `decoded` somehow has a prototype with an `exp` property (which makes the `hasownProperty` check understandable).